_posts/2020-08-26-#19339.md

```diff
@@ -42,7 +42,7 @@ commit: 67bb714
   node’s clients (RPC and wallet).  When a node receives a transaction from its
   peers, it uses an empty value for `nAbsurdFee` when calling ATMP (you can see
   the values of `nAbsurdFee` removed in the [Remove absurdfee from
-  accepttomempool](https://github.com/bitcoin-core-review-club/bitcoin/commit/67bb7148)
+  accepttomempool](https://github.com/bitcoin/bitcoin/commit/67bb7148)
```

Comment:
This should stay pointing to the commit in the bitcoin-core-review-club repo. It's a commit in the branch, so it might not be added to master, and therefore might be garbage collected from bitcoin/bitcoin at some point.

Reply:
thanks -- done